SU-SWS/react_paragraphs
# Filter Row order to get rid of null values

# READY FOR REVIEW

# Summary
- When a row doesn't exist, the rowOrder array has `null` values in the array. then sorting breaks because of the `null` value. so filter those out
- After initial construction, set the field value so that if a user doesn't make any changes, the data will be preserved.

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. checkout this branch and clear browser caches
1. create a basic page with at least 3 rows
1. programatically delete a middle item. easiest way is to delete a row in the database
1. Go back to the page and try to reorder the item.
1. verify it doesnt break.
1. click to edit the page but dont touch anything
1. click to save and verify you still have all the paragraphs

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
